src/index.ts

```diff
@@ -85,44 +82,44 @@ export function resolveBundle(
 
 				// Now that we have a resolver, add the new files needed to the stream
 				resolver
-					.resolve(bundle)
-					.then(additionalItems => {
-						return Promise.map(additionalItems, (file: FileInfo) => {
-							pack.entry({ name: file.name, size: file.size }, file.contents);
-							if (file.name === 'Dockerfile') {
-								return bundle.callDockerfileHook(file.contents.toString());
+				.resolve(bundle)
+				.then(additionalItems => {
+					return Promise.map(additionalItems, (file: FileInfo) => {
+						pack.entry({ name: file.name, size: file.size }, file.contents);
+						if (file.name === 'Dockerfile') {
+							return bundle.callDockerfileHook(file.contents.toString());
+						}
+					})
+					.then(() => {
+						return Promise.try(() => {
+							if (resolver.name === 'Standard Dockerfile') {
```

Comment:
I'd never really considered the pluggable aspects when adding the dockerfile hook, good point! I've added an issue: https://github.com/resin-io-modules/resin-bundle-resolve/issues/16